RevenueCatUI/CustomerCenter/ViewModels/CustomerCenterViewModel.swift

```diff
@@ -121,6 +121,10 @@ import Foundation
         configuration?.support.displayVirtualCurrencies == true
     }
 
+    var shouldShowUserDetailsSection: Bool {
+        configuration?.support.displayUserDetailsSection == true
```

Comment:
I'm just wondering, this would default to false in case the `configuration` or `displayUserDetailsSection` are null... probably shouldn't happen but wondering if we should default to true in those cases?